server/utils/presenterUtils.test.ts

```diff
@@ -237,6 +238,82 @@ describe(PresenterUtils, () => {
     })
   })
 
+  describe('durationValue', () => {
+    describe('hours, minutes values', () => {
+      describe('when the model has a null value for the property', () => {
+        describe('and there is no user input data', () => {
+          it('returns empty strings', () => {
+            const utils = new PresenterUtils(null)
+            const value = utils.durationValue(null, 'duration', null)
+
+            expect(value).toMatchObject({ hours: { value: '' }, minutes: { value: '' } })
+          })
+        })
+      })
+
+      describe('when the model has a non-null value for the property', () => {
+        describe('and there is no user input data', () => {
+          it('returns the corresponding values from the model', () => {
+            const utils = new PresenterUtils(null)
+            const value = utils.durationValue(Duration.minutes(75), 'duration', null)
+
+            expect(value).toMatchObject({ hours: { value: '1' }, minutes: { value: '15' } })
+          })
+        })
+      })
+
+      describe('when there is user input data', () => {
+        it('returns the user input data', () => {
+          const utils = new PresenterUtils({ 'duration-hours': 'egg', 'duration-minutes': 7 })
```

Comment:
When I first saw this, I was a little confused about why 'egg' was here, as I assumed it would be doing something with validation errors. Is it just to show that _any_ value can go here?

Reply:
Yep, it's to show that whatever the user input data is, it'll get played back without any attempt to convert it into a number. Does that make sense?

Reply:
👍